### PR TITLE
RGR-693 Implements the File message

### PR DESCRIPTION
**Pull request by elonmitchell**
Problem
=======
- Roger Chat App not supporting attachments anymore in build 217

[link to JIRA RGR-693](https://usxtech.atlassian.net/browse/RGR-693)

Solution
========
- Implemented the File message as the attachment
- It's using the S3 Bucket as the storage

Screenshots (optional):
-----------------------
Screenshot of Android device

### CODE REVIEW

File: src/components/Chat.tsx [hunk 1]
```diff
@@ -197,10 +206,21 @@ const Chat = (props: VariantChatProps): ReactElement => {
...
-      const newMessage = sendMessages[0].text;
-      sendMessage(newMessage);
+
+      let newMessage = null;
+      if (messageType === FreshchatMessageType.Text) {
+        newMessage = [{ text: { content: messageData[0].text } }];
+      } else if (messageType === FreshchatMessageType.File) {
+        newMessage = [{ text: { content: messageData.uri } }];
+      } else if (messageType === FreshchatMessageType.Image) {
+        // Image
```
Comment (miked49er) [inline src/components/Chat.tsx:218]: Do we not need to do anything for Images?

File: src/components/Chat.tsx [hunk 2]
```diff
@@ -197,10 +206,21 @@ const Chat = (props: VariantChatProps): ReactElement => {
...
 
   const handleSend = useCallback(
-    (sendMessages: IMessage[] = []) => {
+    (messageType: FreshchatMessageType, messageData: any) => {
       Keyboard.dismiss();
-      const newMessage = sendMessages[0].text;
-      sendMessage(newMessage);
+
+      let newMessage = null;
+      if (messageType === FreshchatMessageType.Text) {
```
Comment (miked49er) [inline src/components/Chat.tsx:213]: (Stylistic choice) Why not use a switch instead of if, else if 🎨

File: src/components/Chat.tsx (hunk 1 above)
Comment (elonmitchell) [reply src/components/Chat.tsx]: Image needs to be later.

File: src/components/Chat.tsx (hunk 2 above)
Comment (elonmitchell) [reply src/components/Chat.tsx]: I prefer if / else if for a few cases.

**Review by miked49er — APPROVED** [review]

**Review by antonyermolayev — APPROVED** [review]

**Pull request merged**